src/store/selectors/subjectsPicker.js

```diff
@@ -1,53 +1,19 @@
 const subjectsPicker = ({ primary, lingual }) => {
-  const maxScoreWrapper = {
-    primary: {},
-    lingual: {}
-  }
-
-  let maxPrimaryScore = 0
-  let maxLingualScore = 0
-  let maxPrimarySubject
-  let maxLingualSubject
-  let emptySubject = {
-    name: '',
-    max: {
-      score: 0,
-      formula: ''
-    }
-  }
-
-  maxPrimarySubject = maxLingualSubject = emptySubject
+  const maxScoreWrapper = {}
```

Comment:
I very like the idea of sorting and finding the best or two best subjects. I think I can implement it and it's still very clear to me. However this doesn't return the empty subject as it was in my code (the components rely on max.score and name properties). They check for instance maxScoreWrapper.primary.max.score, but when we got no primary subject, primary property is undefined.